CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 Change log
 -----------
 
+* (backport) kernel-modules-headers: specify cross compiler command [Florin]
```

Comment:
Because it is a change we did a few months ago and it went directly into 2.X
In the meanwhile things have advance in 1.X and only now we add these in this 1.X branch so it's not exactly at the same time the patches were developed so I thought it brings a little bit more context when somebody will look into it and understand that it is a feature which was brought into this branch long after it was developed.

But I can remove this and will do it